.travis.yml

```diff
@@ -166,6 +166,33 @@ jobs:
         - cd $TRAVIS_BUILD_DIR/../metis-4.0;
           mv libmetis.a ..; rm -rf *; mv ../libmetis.a .
 
+    - os: linux
+      compiler: gcc
+      name: "Linux: Parallel (cmake)"
+      addons:
+        apt:
+          packages:
+            - mpich
+            - libmpich-dev
+      env: MPI=YES
+      script:
+        - cd ${TRAVIS_BUILD_DIR}
+        - mkdir ${TRAVIS_BUILD_DIR}/build
+        - cd ${TRAVIS_BUILD_DIR}/build
+        - ls ${TRAVIS_BUILD_DIR}/../metis-4.0
+        - cmake .. -DMFEM_USE_MPI=ON -DHYPRE_DIR=${TRAVIS_BUILD_DIR}/../hypre-2.10.0b/src/hypre -DMETIS_INCLUDE_DIRS=${TRAVIS_BUILD_DIR}/../metis-4.0/Lib -DMETIS_LIBRARIES=${TRAVIS_BUILD_DIR}/../metis-4.0/libmetis.a
```

Comment:
DMETIS_**INCLUDE**_DIRS=${TRAVIS_BUILD_DIR}/../metis-4.0/**Lib**
Is that really the case?

Reply:
That's where the headers are with METIS 4. However, we do not actually include any of those headers in MFEM -- we just copied the prototypes of the functions we use, e.g. in `mesh.cpp`. We only really need the METIS include path when using METIS 5.

On the other hand, in the CMake build system we (probably) expect `*_INCLUDE_DIRS` to be defined for every library that is enabled.

Reply:
@jandrej @v-dobrev 
After further investigation, both `METIS_INCLUDE_DIRS` and `METIS_LIBRARIES` are unnecessary here. In fact `metis-4.0` is at the default location, so `FindMetis` will find it and configure it.

I like to see CI builds as a demonstrator of "how to build MFEM" though. So I would agreed with specifying `METIS_DIR`, which seems the most appropriate way to manually specify a location for metis in this project. 

Does it sound reasonable?

To sum-up, I suggest either:
- completely removing `-DMETIS_INCLUDE_DIRS=${TRAVIS_BUILD_DIR}/../metis-4.0/Lib -DMETIS_LIBRARIES=${TRAVIS_BUILD_DIR}/../metis-4.0/libmetis.a`
- or replacing them with `-DMETIS_DIR=${TRAVIS_BUILD_DIR}/../metis-4.0`

Reply:
I'm fine with both choices, with a slight preference for the second one.